config/elasticsearch/mappings/users.json

```diff
@@ -59,6 +59,28 @@
     },
     "username": {
       "type": "keyword"
+    },
+    "profile_fields": {
```

Comment:
@mstruve I think the only things we're really interested in are field names and values, the ID and group id aren't something we search by.

Reply:
Do users or admins have the ability to change the name of profile fields? If they do then we are going to want to make sure we store and search by the ID otherwise every time a profile field name is updated we would have to go and find all users with that profile field old name and update them.

Reply:
Admins have the ability to change them, BUT the attribute name will never change, exactly for this reason. Say you create a profile field with a description of "Twitter Handle", the attribute will automatically be called "twitter_handle". If you later rename it to "Twitter nickname", the display in the forms will change, but never the attribute. We never directly expose the attribute to admins, so they can't change it.

Reply:
I found the following:

* `20200305201627_index_users_to_elasticsearch.rb`: already commented out
* `20200406213152_re_index_users_to_elasticsearch.rb`: already commented out
* `20200519142908_re_index_feed_content_and_users_to_elasticsearch.rb`: I now commented out the section about users and added a link to this discussion.

I also updated my current script to use the `BulkIndexWorker`, like so:

```ruby
      User.select(:id).in_batches(of: 200) do |batch|
        Search::BulkIndexWorker.set(queue: :default).perform_async("User", batch.ids)
      end
```

I wonder if there is a way to make data update scripts that deal with ES idempotent, going back and commenting out old ones does not seem like the right thing to do long term.

Reply:
Ah description is what everyone sees and we can edit, I like that! 

> I wonder if there is a way to make data update scripts that deal with ES idempotent, 

I'll put some thought against that.